udata_front/theme/gouvfr/templates/dataset/resource/card.html

```diff
@@ -3,7 +3,7 @@
 {% set available = resource.extras['check:status']
     and resource.extras['check:status'] >= 200 and resource.extras['check:status'] < 400 %}
 {% set unavailable = resource.extras['check:status'] and resource.extras['check:status'] >= 400 %}
-{% set last_update = resource.published if resource.published > resource.modified else resource.modified %}
+{% set last_update = resource.modified %}
```

Comment:
Same comment: We should take harvest into account (may be done in https://github.com/etalab/udata-front/pull/201)

Reply:
done in [#201](https://github.com/etalab/udata-front/pull/201)